collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 25
+update_every = 1
```

Comment:
And not sure about `update_every`, do we really need it? If i remember it correctly we use `10` as minimal `every` for stock alarms.

@thiagoftsm is it ok to do `http://127.0.0.1:19999/api/v1/alarms?all` every second? I mean technically - it could be implemented not multithreaded and web thread would block health thread - all this stuff.

Reply:
Set update_every to be 5 by default. Happy to set it higher if we want.

Reply:
See, minimum is 10, i think it makes sense to have `update_every` == 10 by default.

```cmd
0 health.d (nvidia-smi_web_info)$ grep "every: " *
adaptec_raid.conf:   every: 10s
adaptec_raid.conf:   every: 10s
am2320.conf:   every: 10s
apache.conf:   every: 10s
apcupsd.conf:   every: 1m
apcupsd.conf:   every: 60s
apcupsd.conf:   every: 10s
backend.conf:   every: 1m
backend.conf:   every: 10s
backend.conf:   every: 10s
backend.conf:   every: 10s
backend.conf:#   every: 10s
bcache.conf:   every: 1m
bcache.conf:   every: 1m
beanstalkd.conf:   every: 10s
beanstalkd.conf:#   every: 10s
beanstalkd.conf:#   every: 10s
bind_rndc.conf:   every: 60
boinc.conf:   every: 1m
boinc.conf:   every: 1m
boinc.conf:   every: 1m
boinc.conf:   every: 1m
btrfs.conf:   every: 10s
btrfs.conf:   every: 10s
btrfs.conf:   every: 10s
btrfs.conf:   every: 10s
ceph.conf:   every: 10s
cgroups.conf:   every: 1m
cgroups.conf:   every: 10s
cgroups.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
cockroachdb.conf:   every: 10s
couchdb.conf:   every: 10s
cpu.conf:   every: 1m
cpu.conf:   every: 1m
cpu.conf:   every: 5m
cpu.conf:   every: 1m
dbengine.conf: every: 10s
dbengine.conf: every: 10s
dbengine.conf: every: 10s
dbengine.conf: every: 10s
disks.conf:   every: 1m
disks.conf:   every: 1m
disks.conf:   every: 1m
disks.conf:   every: 10s
disks.conf:   every: 1m
disks.conf:   every: 10s
disks.conf:   every: 1m
disks.conf:   every: 1m
dns_query.conf:   every: 10s
dnsmasq_dhcp.conf:   every: 10s
dockerd.conf:   every: 10s
elasticsearch.conf:   every: 10s
entropy.conf:   every: 5m
exporting.conf:   every: 10s
exporting.conf:   every: 10s
exporting.conf:   every: 10s
fping.conf:   every: 10s
fping.conf:   every: 10s
fping.conf:   every: 10s
fping.conf:   every: 10s
fronius.conf:   every: 10s
gearman.conf:   every: 10s
gearman.conf:   every: 10s
haproxy.conf:   every: 10s
haproxy.conf:   every: 10s
haproxy.conf:   every: 10s
hdfs.conf:   every: 10s
hdfs.conf:   every: 10s
hdfs.conf:   every: 10s
hdfs.conf:   every: 10s
hdfs.conf:   every: 10s
hdfs.conf:   every: 10s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 5s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 10s
httpcheck.conf:   every: 30s
httpcheck.conf:   every: 10s
ioping.conf:   every: 10s
ipc.conf:   every: 10s
ipc.conf:   every: 10s
ipfs.conf:   every: 10s
ipmi.conf:   every: 10s
ipmi.conf:   every: 10s
isc_dhcpd.conf:   every: 60
kubelet.conf:      every: 10s
kubelet.conf:      every: 10s
kubelet.conf:      every: 10s
kubelet.conf:   every: 10s
kubelet.conf:   every: 10s
kubelet.conf:   every: 10s
kubelet.conf:   every: 10s
kubelet.conf:   every: 10s
kubelet.conf:   every: 10s
lighttpd.conf:   every: 10s
linux_power_supply.conf:   every: 10s
load.conf:   every: 1m
load.conf:   every: 1m
load.conf:   every: 1m
load.conf:   every: 1m
mdstat.conf:   every: 10s
mdstat.conf:   every: 10s
mdstat.conf:   every: 10s
mdstat.conf:   every: 10s
megacli.conf:   every: 10s
megacli.conf:   every: 10s
megacli.conf:   every: 10s
megacli.conf:   every: 10s
megacli.conf:   every: 10s
memcached.conf:   every: 10s
memcached.conf:   every: 10s
memcached.conf:   every: 1m
memcached.conf:   every: 10s
memory.conf:   every: 1m
memory.conf:   every: 1m
memory.conf:   every: 10s
mongodb.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
mysql.conf:   every: 10s
named.conf:   every: 10s
net.conf:    every: 10s
net.conf:    every: 10s
net.conf:    every: 10s
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 1m
net.conf:   every: 10s
net.conf:   every: 10s
netfilter.conf:   every: 10s
netfilter.conf:   every: 10s
nginx.conf:   every: 10s
nginx_plus.conf:   every: 10s
phpfpm.conf:   every: 10s
pihole.conf:   every: 10s
pihole.conf:   every: 10s
pihole.conf:   every: 10s
pihole.conf:   every: 10s
pihole.conf:   every: 10s
portcheck.conf:   every: 10s
portcheck.conf:   every: 5s
portcheck.conf:   every: 10s
portcheck.conf:   every: 10s
postgres.conf:   every: 10s
processes.conf:   every: 5s
pulsar.conf:   every: 10s
qos.conf:#   every: 30s
ram.conf:   every: 10s
ram.conf:   every: 10s
ram.conf:   every: 10s
ram.conf:   every: 10s
ram.conf:   every: 10s
redis.conf:   every: 10s
redis.conf:   every: 10s
redis.conf:   every: 10s
retroshare.conf:   every: 10s
retroshare.conf:   every: 1m
riakkv.conf:   every: 10s
riakkv.conf:   every: 10s
riakkv.conf:   every: 30s
riakkv.conf:   every: 10s
riakkv.conf:   every: 30s
riakkv.conf:   every: 10s
riakkv.conf:   every: 10s
scaleio.conf:   every: 10s
scaleio.conf:   every: 10s
scaleio.conf:   every: 10s
softnet.conf:   every: 10s
softnet.conf:   every: 10s
softnet.conf:   every: 10s
squid.conf:   every: 10s
stiebeleltron.conf:   every: 10s
swap.conf:   every: 1m
swap.conf:   every: 10s
swap.conf:   every: 10s
tcp_conn.conf:   every: 10s
tcp_listen.conf:   every: 10s
tcp_listen.conf:   every: 10s
tcp_listen.conf:   every: 10s
tcp_listen.conf:   every: 10s
tcp_mem.conf:   every: 10s
tcp_orphans.conf:   every: 10s
tcp_resets.conf:   every: 10s
tcp_resets.conf:   every: 10s
tcp_resets.conf:   every: 10s
tcp_resets.conf:   every: 10s
tcp_resets.conf:   every: 10s
udp_errors.conf:   every: 10s
udp_errors.conf:   every: 10s
udp_errors.conf:   every: 10s
unbound.conf:   every: 10s
unbound.conf:   every: 10s
unbound.conf:   every: 10s
varnish.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vcsa.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 1m
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vernemq.conf:   every: 10s
vsphere.conf:   every: 20s
vsphere.conf:   every: 20s
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
vsphere.conf:   every: 20s
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
vsphere.conf:   every: 1m
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 30s
web_log.conf:   every: 10s
web_log.conf:   every: 30s
web_log.conf:   every: 30s
web_log.conf:   every: 30s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 10s
web_log.conf:   every: 30s
web_log.conf:   every: 10s
web_log.conf:   every: 30s
web_log.conf:   every: 30s
web_log.conf:   every: 30s
whoisquery.conf:   every: 60s
whoisquery.conf:   every: 60s
wmi.conf:   every: 10s
wmi.conf:   every: 1m
wmi.conf:   every: 10s
wmi.conf:   every: 10s
wmi.conf:   every: 1m
wmi.conf:   every: 1m
wmi.conf:   every: 1m
wmi.conf:   every: 1m
wmi.conf:   every: 10s
x509check.conf:   every: 60s
x509check.conf:   every: 60s
x509check.conf:   every: 60s
zfs.conf:   every: 1m
zookeeper.conf:   every: 10s
```